_posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.3.0"
+authors:
+- kkondaka
+- dlv
+date: 2023-05-25 21:30:00 -0700
+categories:
+  - releases
+meta_keywords: Data Prepper 2.3.0
+meta_description: Data Prepper 2.3.0 with enhancements to expressions, event tagging, enhancements to add_entries processor, s3 sink and tail sampling processor
+---
+
+Data Prepper 2.3.0 is now available for [download](https://opensearch.org/downloads.html#data-prepper)!
+This release introduces a number of changes that help with Data Prepper’s ability to process complex expressions with functions, arithmetic and string operations.
+
+## Enhancements to Data Prepper expressions
+
+DataPrepper 2.3 supports using functions in expressions. List of functions supported are can be found at https://github.com/opensearch-project/data-prepper/blob/main/docs/expression_syntax.md
+
+DataPrepper 2.3 supports three types of expressions
+
+* Conditional expressions
+    * Conditional expressions evaluate to a result of Boolean type. The expressions can now have functions in them. For example, length(/message) > 20 would evaluate to true if length of the message field in the event is greater than 20 other wise it evaluates to false. 
+* Arithmetic expressions
+    * Arithmetic expressions evaluate to a result of Integer or Float type. The expressions can have simple arithmetic operators like `+,-,*,/` with functions or JSON pointers or literals as operands. For example, the following expression will add the length of message field of type string in the event with the value of event metadata with key `integerField` and subtracts 1 from it
+
+ `length(/message) + getMetadata("integerField") - 1`
```

Comment:
This may be better with triple backticks around this. Similar to lines 39-45 below.